docker/scylla-bench/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.18
+
+ARG version
+ENV version=${version:-v0.1.8}
```

Comment:
I think we should not have default for the `scylla-bench`, because if we omit it building an image then we get unexpected/old version and success building image.
So, better to avoid such situation.